qiita_pet/handlers/api_proxy/processing.py

```diff
@@ -80,18 +81,30 @@ def list_options_handler_get_req(command_id, artifact_id=None):
         A dictionary containing the commands information
         {'status': str,
          'message': str,
-         'options': list of dicts of {'id: str', 'name': str,
-                                      'values': dict of {str: str}}}
+         'options': list of dicts of {'id: str',
+                                      'name': str,
+                                      'values': dict of {str: str}},
+         'req_options': dict,
+         'opt_options': dict,
+         'extra_artifacts': dict}
```

Comment:
@antgonza if I understand this right, the first artifact is passed the legacy way and any additional artifacts are stored in here?

Reply:
yup, mainly because the "legacy" is the "main" input and the "extra_artifacts" are other that are available that can be used in the same command.

Reply:
After discussing with @antgonza, I agreed that passing the extra artifacts through a separate channel and not with the main 'returned' artifact was prudent. Since they will be handled differently, it makes sense to keep them separated rather than join them only to subsequently filter them out again.